test/test_storage.py

```diff
@@ -118,63 +119,62 @@ def restore(self, snapshot: typing.Dict[str, typing.Any]) -> None:
         s = Sample(f, 'key')
         f.register_type(Sample, None, Sample.handle_kind)
         s.on.event.emit('foo')
-        self.assertEqual('foo', s.observed_content)
+        assert s.observed_content == 'foo'
         s.on.event.emit(1)
-        self.assertEqual(1, s.observed_content)
+        assert s.observed_content == 1
         s.on.event.emit(None)
-        self.assertEqual(None, s.observed_content)
+        assert None is s.observed_content
```

Comment:
This would read more naturally the other way around:

```suggestion
        assert s.observed_content is None
```

Reply:
I think this might be the result of pytestify since in the original version None is first, lol. I'll do a check to fix all similar issues.